login.html

```diff
@@ -4,13 +4,16 @@
   <meta charset="UTF-8">
   <title>Instamood | Login</title>
   <!-- TODO: Include Bootstrap -->
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <!-- TODO: Link your stylesheet -->
+  <script src="http://code.jquery.com/jquery-latest.js"></script>
```

Comment:
Link your stylesheet so you can customize the page 
`<link rel="stylesheet" href="style.css"`>